fury/actor.py

```diff
@@ -2543,3 +2543,76 @@ def sdf(centers, directions=(1, 0, 0), colors=(1, 0, 0), primitives='torus',
     shader_to_actor(box_actor, "fragment", impl_code=fs_impl_code,
                     block="light")
     return box_actor
+
+
+def parametric_surface(centers, directions=(1, 0, 0), colors=(1, 0, 0),
+                       scales=1, name='mobius_strip', npoints=100):
+    """Visualize one or many parametric surfaces with different features.
+    Centered by default such that their centroids coincide with the origin i.e.
+    their centroids lie on [0, 0, 0].
+    Parameters
```

Comment:
need an empty line between `parameters` and le line above

Reply:
Done